core/arxiv/submission/services/classic/load.py

```diff
@@ -0,0 +1,209 @@
+                version_submission = patch_jref(version_submission, row)
+            # For withdrawals and cross-lists, we want to get data from
+            # deleted rows since we keep track of all requests in the NG
+            # submission.
+            elif row.is_withdrawal():
+                logger.debug('Withdrawal row')
+                # This should update the reason_for_withdrawal (if applied),
+                # and add a WithdrawalRequest to user_requests.
+                version_submission = patch_withdrawal(version_submission, row)
+            elif row.is_crosslist():
+                logger.debug('Crosslist row')
+                # This should update the secondary classifications (if applied)
+                # and add a CrossListClassificationRequest to user_requests.
+                version_submission = patch_cross(version_submission, row)
+
+            # We want hold information represented as a Hold on the submission
+            # object, not just the status.
+            if version_submission.is_on_hold:
+                version_submission = patch_hold(version_submission, row)
+        versions.append(version_submission)
+    submission = copy.deepcopy(versions[-1])
+    submission.versions = [ver for ver in versions if ver and ver.announced]
+    return submission
+
+
+def to_submission(row: models.Submission,
+                  submission_id: Optional[int] = None) -> domain.Submission:
+    """
+    Generate a representation of submission state from a DB instance.
+
+    Parameters
+    ----------
+    row : :class:`.models.Submission`
+        Database row representing a :class:`.domain.submission.Submission`.
+    submission_id : int or None
+        If provided the database value is overridden when setting
+        :attr:`domain.Submission.submission_id`.
+
+    Returns
+    -------
+    :class:`.domain.submission.Submission`
+
+    """
+    status = status_from_classic(row.status)
+    primary = row.primary_classification
+    if row.submitter is None:
+        submitter = domain.User(native_id=row.submitter_id,
+                                email=row.submitter_email)
+    else:
+        submitter = row.get_submitter()
+    if submission_id is None:
+        submission_id = row.submission_id
+
+    license: Optional[domain.License] = None
+    if row.license:
+        label = LICENSES[row.license]['label']
+        license = domain.License(uri=row.license, name=label)
+
+    primary_clsn: Optional[domain.Classification] = None
+    if primary and primary.category:
+        _category = domain.Category(primary.category)
+        primary_clsn = domain.Classification(category=_category)
+    secondary_clsn = [
+        domain.Classification(category=domain.Category(db_cat.category))
+        for db_cat in row.categories if not db_cat.is_primary
+    ]
+
+    content: Optional[domain.SubmissionContent] = None
+    if row.package:
+        if row.package.startswith('fm://'):
```

Comment:
Presumably this has to do with the file management service? A brief comment would be helpful.

Reply:
https://github.com/arXiv/arxiv-submission-core/issues/67